scripts/release.js

```diff
@@ -0,0 +1,45 @@
+const {
```

Comment:
It would be nice to have a prompt at the beginning of the script. In a case when the script was executed accidentally.